### PR TITLE
feat: alby hub connector

### LINKED ISSUE #3261: Add Alby Hub connector
### Feature description

Alby Hub is not featured as an "Own wallet", while it should be. 

Even if it uses NWC connector, other wallets also wrap on existing connector (ie. voltage)

@reneaaron brought this up today

### Describe the solution

image

### Describe alternatives

_No response_

### Additional context

_No response_

### Are you working on this?

None

### PR DESCRIPTION
**Pull request by pavanjoshi914**
### Describe the changes you have made in this PR

specific connector for alby hub
### Link this PR to an issue [optional]

Fixes #3261 

### Type of change

(Remove other not matching type)

- `feat`: New feature (non-breaking change which adds functionality)

### Checklist

- [X] Self-review of changed code
- [X] Manual testing
- [x] Added automated tests where applicable
- [X] Update Docs & Guides
- For UI-related changes
- [X] Darkmode
- [X] Responsive layout

### CODE REVIEW

File: src/extension/background-script/connectors/albyhub.ts
Comment (reneaaron) [inline src/extension/background-script/connectors/albyhub.ts:1]: We don't need a connector, but instead should use the NWC one. Please refer to how the voltage "connector" is implemented.

File: static/assets/icons/albyhub.svg
Comment (reneaaron) [inline static/assets/icons/albyhub.svg:1]: No idea where you have this icon from, but this image has multiple flaws. (rounded corners, wrong colors, wrong borders, etc)

I think we should use sth like this: 
https://github.com/getAlby/hub/blob/master/frontend/public/icon-1024.png

File: src/i18n/locales/en/translation.json [hunk 1]
```diff
@@ -334,6 +334,19 @@
           "connection_failed": "Connection failed. Are your credentials correct?"
         }
       },
+      "albyhub": {
+        "title": "Alby Hub",
+        "page": {
+          "instructions": "Paste a NWC connection string from Alby Hub",
```
Comment (reneaaron) [inline src/i18n/locales/en/translation.json:340]: It's called "connection secret", please also adjust existing items.

File: src/i18n/locales/en/translation.json [hunk 2]
```diff
@@ -334,6 +334,19 @@
...
+          "url": {
+            "label": "NWC connection string from Alby Hub",
+            "placeholder": "nostr+walletconnect://69effe..."
+          },
+          "errors": {
+            "connection_failed": "Connection failed. Is your Alby Hub online and app connection enabled?"
```
Comment (reneaaron) [inline src/i18n/locales/en/translation.json:346]: What does "app connection enabled" mean in that context?

**Review by reneaaron — CHANGES_REQUESTED** [review]

File: src/extension/background-script/connectors/albyhub.ts
Comment (pavanjoshi914) [reply src/extension/background-script/connectors/albyhub.ts:1]: aah okay. i thought we want to keep it separate for any specific feature

File: static/assets/icons/albyhub.svg
Comment (pavanjoshi914) [reply static/assets/icons/albyhub.svg:1]: grabbed that from figma

File: src/i18n/locales/en/translation.json (hunk 1 above)
Comment (pavanjoshi914) [reply src/i18n/locales/en/translation.json]: resolved

File: src/i18n/locales/en/translation.json (hunk 2 above)
Comment (pavanjoshi914) [reply src/i18n/locales/en/translation.json:346]: mean that app connection still exists and usable

**Review by reneaaron — APPROVED** [review]
Added some copy improvements

tACK

**Pull request merged**